x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -380,6 +382,53 @@ export const formatExceptionItemForUpdate = (
   };
 };
 
+/**
+ * Maps "event." fields to "signal.original_event.". This is because when a rule is created
+ * the "event" field is copied over to "original_event". When the user creates an exception,
+ * they expect it to match against the original_event's fields, not the signal event's.
+ * @param exceptionItems new or existing ExceptionItem[]
+ */
+export const prepareExceptionItemsForBulkClose = (
+  exceptionItems: Array<ExceptionListItemSchema | CreateExceptionListItemSchema>
+): Array<ExceptionListItemSchema | CreateExceptionListItemSchema> => {
+  const replaceField = (fieldToReplace: string) => {
+    return fieldToReplace.startsWith('event.')
+      ? fieldToReplace.replace(/^event./, 'signal.original_event.')
+      : fieldToReplace;
+  };
+
+  return exceptionItems.map((item: ExceptionListItemSchema | CreateExceptionListItemSchema) => {
+    if (item.entries !== undefined) {
+      const newEntries = item.entries.map((itemEntry: EntriesArray[0]) => {
+        if (itemEntry.type === 'nested') {
+          const newNestedEntries = itemEntry.entries.map((nestedEntry: EntryMatch) => {
```

Comment:
I think we only want to replace top level entries that start with `event.` so we can remove the whole `if (itemEntry.type === 'nested') {` block and always execute what is currently the `else` part.